yarn.lock

```diff
@@ -29449,4 +29449,4 @@ zwitch@^1.0.0:
 zwitch@^2.0.0:
   version "2.0.2"
   resolved "https://registry.npmjs.org/zwitch/-/zwitch-2.0.2.tgz#91f8d0e901ffa3d66599756dde7f57b17c95dce1"
-  integrity sha512-JZxotl7SxAJH0j7dN4pxsTV6ZLXoLdGME+PsjkL/DaBrVryK9kTGq06GfKrwcSOqypP+fdXGoCHE36b99fWVoA==
+  integrity sha512-JZxotl7SxAJH0j7dN4pxsTV6ZLXoLdGME+PsjkL/DaBrVryK9kTGq06GfKrwcSOqypP+fdXGoCHE36b99fWVoA==
```

Comment:
This looks like the only file that requires `@reviewers` review, and the change is not material?  So I'd recommend reverting this.

Reply:
couldn't understand that, why does it take as change, they are same I guess...

Reply:
Looks like maybe fixed by [this](https://github.com/backstage/backstage/pull/8052), if you rebase?

Reply:
done :)